app/controllers/spotlight/translations_controller.rb

```diff
@@ -19,6 +19,13 @@ def update
 
     private
 
+    def translations_attributes_destroying_blanks
+      exhibit_params.to_h[:translations_attributes].map do |index, attrs|
+        attrs['_destroy'] = true if attrs['value'].blank?
+        [index, attrs]
```

Comment:
I was thinking I could guard against sending the `_destroy` key when there is no `id`, but this seems to workout just fine.

Reply:
Instead of `#map`, maybe `#transform_values`?